features/order/managing_orders/finalizing_order_payment.feature

```diff
@@ -7,6 +7,7 @@ Feature: Finalizing order payment
     Background:
         Given the store operates on a single channel in "United States"
         And the store has a product "Angel T-Shirt"
+        And there is 1 item of product "Angel T-Shirt" available in the inventory
```

Comment:
Why do we need this? I think inventory should be disabled by default. (tracked = false on variant)